index.html

```diff
@@ -2778,9 +2789,11 @@ <h3>Terms of Use</h3>
       "name": "Bachelor of Science and Arts"
     }
     },
-    "proof": { <span class="comment">...</span> }
+    "proof": { 
+
+  }
```

Comment:
-1 to this change, it breaks consistency with the other examples. If you're worrying about `...` being copy-pasted, it won't be... we have CSS rules to prevent it from being copied to the paste buffer.

Reply:
fixed, thanks for pointing out this issue. I've reverted it back to how it originally was.